app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/components/topBar/index.tsx

```diff
@@ -0,0 +1,123 @@
+/** @file The top-bar of dashboard. */
+import { FC, useState } from 'react'
+
+import * as hooks from '../../../hooks'
+import UserMenu from './userMenu'
+
+// ==============
+// === TopBar ===
+// ==============
+
+/** ICONS constant using by TopBar. */
+const ICONS = {
+    BAR: (
+        <svg
+            width="16"
+            height="16"
+            viewBox="0 0 16 16"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+        >
+            <rect x="2" y="1" width="12" height="3" fill="#767676" />
+            <rect x="2" y="6" width="12" height="3" fill="#767676" />
+            <rect x="2" y="11" width="12" height="3" fill="#767676" />
+        </svg>
+    ),
+    SEARCH: (
+        <svg
+            width="16"
+            height="16"
+            viewBox="0 0 16 16"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+        >
+            <g opacity="0.5">
+                <path
+                    d="M11.4142 10L15.6569 14.2426L14.2426 15.6569L10 11.4142L11.4142 10Z"
+                    fill="#3E515F"
+                />
+                <circle cx="7" cy="7" r="5" stroke="#3E515F" stroke-width="2" />
+            </g>
+        </svg>
+    ),
+    CHAT: (
+        <svg
+            width="16"
+            height="17"
+            viewBox="0 0 16 17"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+        >
+            <g opacity="0.9">
+                <ellipse cx="8" cy="8" rx="8" ry="7.5" fill="white" />
+                <path d="M4.17269e-05 16.5L2 10.5L5.50006 14L4.17269e-05 16.5Z" fill="white" />
+            </g>
+        </svg>
+    ),
+}
+/** The BAR icon shadow. */
+const BAR_BOX_SHADOW =
+    '0px 18px 80px rgba(0, 0, 0, 0.11), 0px 7.51997px 33.4221px rgba(0, 0, 0, 0.079074), 0px 4.02054px 17.869px rgba(0, 0, 0, 0.0655718), 0px 2.25388px 10.0172px rgba(0, 0, 0, 0.055), 0px 1.19702px 5.32008px rgba(0, 0, 0, 0.0444282), 0px 0.498106px 2.21381px rgba(0, 0, 0, 0.030926)'
```

Comment:
this specific shadow has been added to tailwind config in another PR:
https://github.com/enso-org/enso/blob/79f54bdb6e8dc6653e8df44be737876baf6d743c/app/ide-desktop/lib/dashboard/tailwind.config.ts#L35

so maybe add a FIXME to change it to `shadow-soft` when that PR gets merged in